media-gfx/sane_backends/sane_backends-1.1.1.recipe

```diff
@@ -41,7 +41,6 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	cmd:python3$secondaryArchSuffix
```

Comment:
AFAICT python doesn't provide a $secondaryArchsuffix, if this is required then it would be:
```
cmd:python3
```

Reply:
Is it required at runtime or build-time? (a bit confused atm) :)

Reply:
Build.

Reply:
Will merge, watch buildmaster on eventual failed builds.